src/poliastro/examples.py

```diff
@@ -58,3 +58,186 @@
     epoch=time.Time("2015-11-05 12:00", scale="utc"),
 )
 """Comet 67P/Churyumov–Gerasimenko orbit example"""
+
+debris_orbits_list = []
```

Comment:
I'd prefer that we put this only in the tests, rather than shipping it with the library.